src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -128,7 +128,10 @@
     <PackageReference Include="NuGet.VisualStudio" Version="3.3.0" />
     <PackageReference Include="System.IO.Abstractions" Version="7.1.10" />
     <PackageReference Include="System.Reflection.Metadata" Version="1.2.0" />
-    <PackageReference Include="Microsoft.VSSDK.BuildTools" Version="16.3.2099" />
+    <PackageReference Include="Microsoft.VSSDK.BuildTools" Version="16.8.3038">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
```

Comment:
I upgraded the version of the build tools to match the one in the `EmbeddedVsix` project.
The UI added the `PrivateAssets` and `IncludeAssets` settings, which are actually what we want - this project will use all of the included assets, but the references won't flow to child projects.